stats/test/platform-routes.test.js

```diff
@@ -79,40 +71,25 @@ describe('Platform Routes HTTP request handler', () => {
       await assertResponseStatus(res, 200)
       const metrics = await res.json()
       assert.deepStrictEqual(metrics, [
-        { day: '2024-01-11', station_id_count: 1 },
-        { day: '2024-01-12', station_id_count: 2 }
+        { day: '2024-01-11', station_count: 3 },
+        { day: '2024-01-12', station_count: 4 }
```

Comment:
I believe the intention of this PR is to change the internal implementation while preserving the behaviour observed from outside. In that case it's best to keep the assertions unchanged, to show that the new implementation returns the same responses as the old one.

On the second look, I see that we are changing the response and renaming `station_id_count` to `station_count`. I think that's a good change, I am happy to land it. But I still think it's better to keep the count values unchanged.

The same comment applies to other tests below that may be changing the returned data.

Reply:
Agree with what you've said here, I'll switch the values back.

It was `station_id_count` before because we were using the dailyDistinctCount method which just appended the column name with "_count", but now I think `station_count` may be better.

The only real consumers of the data today, as far as I'm aware, is Grafana, and we can change the dashboard quite easily to match small naming changes like this. But if you'd prefer this PR to maintain exact external behavior, I can make those changes instead!

Reply:
+1 to what you wrote. Let's use `station_count` please.